EarlGrey/Core/GREYAutomationSetup.m

```diff
@@ -106,7 +106,7 @@ - (void)grey_enableAccessibility {
 #pragma mark - Crash Handlers
 
 // Call only asynchronous-safe functions within signal handlers
-// See definition here: https://www.securecoding.cert.org/confluence/display/seccode/BB.+Definitions
+// See definition here: https://www.securecoding.cert.org/confluence/display/c/BB.+Definitions
```

Comment:
change to https://www.securecoding.cert.org/confluence/display/c/SIG00-C.+Mask+signals+handled+by+noninterruptible+signal+handlers ?